### DISCUSSION TITLE
Keycloak Cache support for Memcached or Redis

### DISCUSSION THREAD

**Discussion by ashishtchaudhari** [keycloak/keycloak#13983] [Q&A]
Hi,
I would like to know if Keycloak support any other mode of caching apart from Infinispan. 
The support for Memcached or Redis will be very helpful as these services are offered by cloud as managed services and can greatly help with the ease of deployment and maintenance of large deployments.

Regards,
Ashish Choudhari

**Comment by thebestgin** [comment]
Good Idea. I would appreciate it too.

**Comment by thebestgin** [comment]
I ask the same feature again in https://github.com/keycloak/keycloak/issues/24849#issue-2002211743

**Comment by stianst** [comment]
We do not plan to add support to Redis directly, see https://github.com/keycloak/keycloak/discussions/48577 for more details.
